subscriber.rb

```diff
@@ -9,12 +10,21 @@ class Subscriber
   def initialize(options)
     @sqs = AwsSQSClient.new.client
     @queue_url = options[0].chomp
-    @sleep_period = valid_sleep_period?(options[1]) ? options[1].to_i : 5
+    @sleep_period = validate_sleep_period(options[1])
   end
 
-  def valid_sleep_period?(sleep_period)
+  def validate_sleep_period(sleep_period)
     max_sleep_period = /\A([3-9]|1[0-5])\z/
-    !sleep_period.nil? && sleep_period.match?(max_sleep_period)
+    begin
+      if !sleep_period.nil? && sleep_period.match(max_sleep_period)
```

Comment:
I'd prefer that you have another error being raised indicating that `sleep_period` is mandatory, and not keep it as a `QItArgumentError` which implies that an argument has been provided and is an incorrect one.